app/controllers/course/forum/posts_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Course::Forum::PostsController < Course::Forum::Controller
+  include Course::Discussion::PostsConcern
+
```

Comment:
So you mean that I should use `authorize_resources :topic` just after including the concern?